mobile/src/containers/settings/Support.jsx

```diff
@@ -15,6 +17,26 @@ export const Support = ({ t, analytics, setAnalytics }) => (
         id: 'analytics_checkbox',
         checked: analytics,
         onChange: setAnalytics
+      },
+      {
+        type: ELEMENT_BUTTON,
+        display: isDebug,
+        title: t('mobile.settings.support.logs.title'),
+        description: t('mobile.settings.support.logs.description'),
+        text: t('mobile.settings.support.logs.button'),
+        theme: 'secondary',
+        onClick: async () => {
+          if (isOnline()) {
+            try {
+              await logInfo(t('mobile.settings.support.logs.title'), true)
+              success()
+            } catch (e) {
+              failure()
```

Comment:
For now all logInfo are success, but we refactor `reporter` when we are more time.